plugin/src/main/groovy/com/btkelly/gnag/utils/ViolationFormatter.java

```diff
@@ -59,13 +61,21 @@ public static String getHtmlStringForAggregatedComment(@NotNull final Violation
         final HtmlStringBuilder builder = new HtmlStringBuilder();
 
         appendViolationNameToBuilder(violation, builder);
-        appendRelativeFilePathToBuilderIfPresent(violation, builder);
-        appendViolationLineNumberToBuilderIfPresent(violation, builder);
+        appendLocationInformationToBuilderIfPresent(violation, builder);
         appendViolationCommentToBuilderIfPresent(violation, builder);
 
         return builder.toString();
     }
 
+    private static void appendViolationReporterToBuilder(
+            @NotNull final Violation violation,
+            @NotNull final HtmlStringBuilder builder) {
+
+        builder.appendBold(VIOLATION_REPORTER_LABEL);
+        builder.append(violation.getReporterName());
+        builder.insertLineBreak();
```

Comment:
FYI: Line break positioning is pretty fragile right now; it works for the two use-cases we care about, but if we ever change the info reported this is a potential hazard.